test/services/debug-service.ts

```diff
@@ -214,7 +215,8 @@ describe("debugService", () => {
 					const debugData = getDebugData();
 					const url = await debugService.debug(debugData, null);
 
-					assert.deepEqual(url, fakeChromeDebugUrl);
+					assert.deepEqual(url.url, fakeChromeDebugUrl);
```

Comment:
why not asserting the whole object? Currently the object may have additional properties and the test will pass. I prefer asserting equality of the whole object, so in case we add new properties, we'll have to add/update tests for them.